test_gradient.py

```
@@ -0,0 +1,119 @@
+# Test gradient
+print("\nTesting gradient...")
+
+
+def logpdf_wrapper(params_dict, data):
+    return likelihood.evaluate(params_dict, data)
+
+
+grad_fn = jax.grad(logpdf_wrapper)
+try:
+    grad = grad_fn(test_params, {})
+    print(f"Gradient computed successfully")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)